### DISCUSSION TITLE
Upload spectrograms to image host

### DISCUSSION THREAD

**Discussion by RogueOneEcho** [RogueOneEcho/caesura#68] [Ideas]
Some users include links to uploaded spectrogram files in the description. 

There's a benefit in doing this for FLAC sources but likely doing so for transcodes serves little purpose and puts a not insignificant cost on the image host.

However, it could be useful for #60

**Comment by k18ldp** [comment]
Other than #59  I also see no benefit of having this

**Comment by RogueOneEcho** [comment]
Closing as it doesn't seem worthwhile and there are already too many open ideas that do have merit.

Also worth noting:
- ptpimg has closed which was the most likely hosting candidate.
- If upload is automated then users are likely to just upload specs that are never even looked at which wastes space on the host. Whereas requiring manual intervention at least puts the onus on the uploader.

Feel free to suggest reasons in favor of re-opeing though.
